Source/WebInspectorUI/UserInterface/Views/FontDetailsPanel.js

```diff
@@ -76,7 +76,10 @@ WI.FontDetailsPanel = class FontDetailsPanel extends WI.StyleDetailsPanel
                 row.value = this._formatPropertyValue(propertyName, fontProperty.value);
 
                 if (propertyName === "font-style")
-                    row.warningMessage = this.nodeStyles.computedPrimaryFont?.synthesizedOblique ? WI.UIString("Font was synthesized to be oblique because no oblique font is available.", "A warning that is shown in the Font Details Sidebar when the font had to be synthesized to support the provided style.") : null;
+                    if (this.nodeStyles.computedPrimaryFont?.synthesizedOblique)
+                        row.warningMessage = fontProperty.value === "italic" ? WI.UIString("Font was synthesized to be italic because no italic font is available.", "A warning that is shown in the Font Details Sidebar when the font had to be synthesized to support the provided style.") : WI.UIString("Font was synthesized to be oblique because no oblique font is available.", "A warning that is shown in the Font Details Sidebar when the font had to be synthesized to support the provided style.");
```

Comment:
NIT: we may want this to be a 
```
if (fontProperty.value === "italic")
    row.warningMessage = ...
else
    row.warningMessage = ...
```
just so that it's not such a long line :)

Reply:
Aren't the italic string is still wrong? In both cases oblique-ness is synthesized, regardless of what was requested IIRC. I'd think `"Font was synthesized to be oblique because no italic font is available."` would be more accurate?

Reply:
> Isn't the italic string still wrong? In both cases oblique-ness is synthesized, regardless of what was requested IIRC. I'd think `"Font was synthesized to be oblique because no italic font is available."` would be more accurate?

You're right! We discussed this exact thing and I forgot. 
Updated the warning message for `italic`.

<img width="572" alt="Screenshot 2023-03-31 at 16 09 32" src="https://user-images.githubusercontent.com/105809798/229145563-8c9811fc-7336-40d4-90b9-67d1f0af63f0.png">